### ISSUE TITLE
Hud is not shown in full screen mode !

### ISSUE THREAD

**Issue by Meseery** [kevin-hirsch/KVNProgress#27]
when i set  ` _fullScreen = YES`  , the hud is not shown on a collection view, but it appears on others !

i used method line : `  [KVNProgress showErrorWithStatus:@"In Progress "] `

**Comment by kevin-hirsch** [comment]
Hello @Meseery,

Can you be more clear on what you expected and what you get?
Do you use the `fullscreen` this way:

   ```objc
KVNProgressConfiguration *configuration = [KVNProgressConfiguration defaultConfiguration];
configuration.fullScreen = YES;

[KVNProgress setConfiguration:configuration];
   ```
?

Also, you say the HUD is not shown on a collection view but you are using the method `showErrorWithStatus:` which displays the HUD on the window not on a certain view like `showErrorWithStatus:onView:`

**Comment by Meseery** [comment]
Hello @kevin-hirsch 
Yes, i used the configuration above, `configuration.fullscreen=YES` and it doesn't work on main view which contains a collection view embedded in scroll view, I used all forms of methods and eventually it doesn't work !

**Comment by kevin-hirsch** [comment]
You are calling the HUD inside the code of your collection view?
Are you using it on other parts of your app and do it work correctly?
So, how and where the HUD is showing? Could you provide a screenshot of your issue?

**Comment by Meseery** [comment]
No, i called it on loading the view on fetching the data and yes, it works in other views ,it's not working in `application:didFinishLaunchingWithOptions:` and it's not shown !

**Comment by Meseery** [comment]
Here're some details:
- i've a `UITabbarController` which have 5 viewcontrollers.
- The home view controller which i set by calling `setSelectedIndex:` method.
-This home view controller have the method `showWithStatus:` in its `viewDidLoad` method.

When loading this view controller, the HUD is not shown, however, it's been shown on other viewcontrollers !!

**Comment by kevin-hirsch** [comment]
So calling `[KVNProgress showErrorWithStatus:@"In Progress"]` is not working in `application:didFinishLaunchingWithOptions:` method from your `AppDelegate` and also from the `viewDidLoad` of the first `UIViewController` contained in your `UITabbarController` at app launch?
I will try to reproduce it and fix it ;)

**Comment by Meseery** [comment]
Yes, exactly ,Thanks :)
